python/lsst/jointcal/testUtils.py

```diff
@@ -111,6 +112,8 @@ def createFakeCcdImage(butler, visit, ccdId, num, instFluxKeyName,
         Value to set for calibrationMean in the created PhotoCalib.
     photoCalibErr : `float`, optional
         Value to set for calibrationErr in the created PhotoCalib.
+    fakeCcdId : `int`, optional
+        If set, use this as the ccdId in the returned CcdImage.
```

Comment:
Consider removing the "If set". Presumably the default value of 12 gets used instead of being a magic number for "unspecified"?